README.md

```diff
@@ -84,6 +84,9 @@ Both [HLSL](https://github.com/Microsoft/DirectXShaderCompiler) and [GLSL](https
 have compilers to target SPIR-V. All of these shader languages can be used with any backend, we
 will handle all of the conversion. Additionally, support for these shader inputs is not going away.
 
+The WGSL Spec is still in developement and full conformity with the current spec is unlikely. 
+[Below](#tracking-the-evolving-wgsl-spec) we elaborate a bit more.
```

Comment:
Specifically here

Reply:
You're right - thanks. Fixed in 94ce7639.